flexmeasures/utils/app_utils.py

```diff
@@ -4,17 +4,56 @@
 
 import click
 from flask import Flask
-from flask.cli import FlaskGroup
+from flask.cli import FlaskGroup, with_appcontext
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.rq import RqIntegration
+from pkg_resources import get_distribution
 
 from flexmeasures.app import create as create_app
 
 
 @click.group(cls=FlaskGroup, create_app=create_app)
+@with_appcontext
 def flexmeasures_cli():
-    """Management scripts for the FlexMeasures platform."""
+    """
+    Management scripts for the FlexMeasures platform.
+    We use @app_context here so things from the app setup are initialised
+    only once. This is crucial for Sentry, for example.
+    """
     pass
 
 
+def init_sentry(app: Flask):
+    """
+    Configure Sentry.
+    We need the app to read the Sentry DSN from configuration, and also
+    to send some additional meta information.
+    """
+    sentry_dsn = app.config.get("SENTRY_DSN")
+    if not sentry_dsn:
+        app.logger.info(
+            "[FLEXMEASURES] No SENTRY_DSN setting found, so initialising Sentry cannot happen ..."
+        )
+        return
+    app.logger.info("[FLEXMEASURES] Initialising Sentry ...")
+    sentry_sdk.init(
+        dsn=sentry_dsn,
+        integrations=[FlaskIntegration(), RqIntegration()],
+        debug=app.debug,
+        release=f"flexmeasures@{get_distribution('flexmeasures').version}",
```

Comment:
Might be useful to also add which plugins are loaded (incl. their version). I'm not sure whether this is the right place to put such info, but this is where the thought came up for me.

Reply:
Looks like this is already handled in line 146.